<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="califp_button">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>30</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Please select the filepath of Calibration photos</string>
    </property>
   </widget>
   <widget class="QLabel" name="Error_show">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>630</y>
      <width>61</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Error:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="error_double">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>630</y>
      <width>311</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="k1_show">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>430</y>
      <width>27</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>k1:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="k1_double">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>430</y>
      <width>328</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fx_show">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>270</y>
      <width>30</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fx:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="fx_double">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>270</y>
      <width>328</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="fy_double">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>310</y>
      <width>328</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fy_show">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>310</y>
      <width>28</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cx_show">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>350</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cx:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cx_double">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>350</y>
      <width>328</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cy_show">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>390</y>
      <width>28</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cy:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cy_double">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>390</y>
      <width>328</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="k2_double">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>470</y>
      <width>328</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="k2_show">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>470</y>
      <width>27</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>k2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="p1_show">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>510</y>
      <width>27</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>p1:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="p1_double">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>510</y>
      <width>328</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="p2_double">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>550</y>
      <width>328</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="p2_show">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>550</y>
      <width>27</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>p2:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="k4_double">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>590</y>
      <width>328</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="k4_show">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>590</y>
      <width>27</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>k4:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="califp_char">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>100</y>
      <width>402</width>
      <height>66</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="califp_show">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>100</y>
      <width>80</width>
      <height>66</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FilePath:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="height_double">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>180</y>
      <width>170</width>
      <height>44</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="width_show">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>180</y>
      <width>57</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="height_show">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>180</y>
      <width>65</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="width_double">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>169</width>
      <height>44</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Start_Calib_button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Start_Writeyaml_button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Write Result to File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menucal">
    <property name="title">
     <string>Camera Calibration</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuStereo_VO">
    <property name="title">
     <string>Stereo VO</string>
    </property>
   </widget>
   <addaction name="menucal"/>
   <addaction name="menuStereo_VO"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
